cmd/vic-machine/configure/configure.go

```diff
@@ -92,10 +93,11 @@ func (c *Configure) Flags() []cli.Flag {
 	compute := c.ComputeFlags()
 	iso := c.ImageFlags(false)
 	debug := c.DebugFlags()
+	cNetwork := c.cNetworks.CNetworkFlags(false)
 
 	// flag arrays are declared, now combined
 	var flags []cli.Flag
-	for _, f := range [][]cli.Flag{target, id, compute, iso, proxies, util, debug} {
+	for _, f := range [][]cli.Flag{target, id, compute, iso, proxies, util, debug, cNetwork} {
```

Comment:
the command help will show options in the slice order, so it will be better to keep it's original order, instead of add all network options in the end.